content/self-host/client-deployment/_index.en.md

```diff
@@ -114,6 +114,9 @@ rustdesk.exe --silent-install
 timeout /t 20
 
 cd "C:\Program Files\RustDesk\"
+rustdesk.exe --install-service -wait -Verbose
```

Comment:
Are you sure this works?

Reply:
I believe `-wait -Verbose` does nothing here.

Reply:
it does install the service yes. you are right sorry just copied it from notes from helping someone earlier and didnt check

Reply:
changed now